packages/create-flex-plugin/src/index.ts

```diff
@@ -1,2 +1,7 @@
+import { notify as updateNotify } from 'flex-dev-utils/dist/updateNotifier';
+
+// Check for update
```

Comment:
this comment seems unnecessary or make it even more explicit?

Reply:
actually `updateNotify` just seems like it should be named something else 😂

Reply:
Renamed to `checkForUpdate`